lib/oli_web/components/delivery/students/students.ex

```diff
@@ -231,7 +231,7 @@ defmodule OliWeb.Components.Delivery.Students do
 
   defp maybe_filter_by_card(students, :low_proficiency, filter_by) do
     Enum.filter(students, fn student ->
-      Metrics.proficiency_range(student.overall_proficiency) == "Low" ||
+      student.overall_proficiency == "Low" ||
```

Comment:
This was a bit strange to me that `student.overall_proficiency` was being run through `Metrics.proficiency_range` again, when it should already be a value of. "Low", "Medium", "High", or "Not enough data" at this point. Further, the way this function is pattern matched, if an invalid value was being passed to `Metrics.proficiency_range`, it would simply return "High" which seems like a bug

Reply:
Nice catch!